CHANGELOG.md

```diff
@@ -1,10 +1,32 @@
 # Changelog
 ## unreleased
 - cleanup dispatcher rules (restrict)
+- introduce subsidiedatabank frontend
+- Add subsidiedatabank graph populated by migrations & db-cleanup
 - update subsidies producer config 
 ### Deploy instructions
+`git pull`
+task: frontend-subsidiedatabank override:
+```
+  frontend-subsidiedatabank:
```

Comment:
Can you make a difference between what needs to happen on dev/qa and prod?
Also, for prod e.g. we need to configure the proxy to redirect the DNS to another machine!